### ISSUE TITLE
Siri Commands

### ISSUE THREAD

**Issue by rv10guy** [nfarina/homebridge-sonos#1]
What Siri commands can be used with this plug-in?

**Comment by sooth** [comment]
Yes, a list of possible commands would be nice. I cannot figure out how to tell Siri to change the volume.

**Comment by schmittx** [comment]
It seems like you can only turn ON/OFF (i.e. "Siri, turn ON/OFF the <NAME>") and set the volume to XX% (i.e. "Siri, set the volume of the <NAME> to XX%), where <NAME> corresponds to "name" in config.json.

**Comment by eddiemoy718** [comment]
I finally got mine to recognize my Speakers, I had the room case not matching.  You need to do an edit for the rooms to see the actual case...

Anyone know what we can do with this besides turn it on and off?

**Comment by vanhoutem** [comment]
The "turn on" and "turn off" commands seem to start and stop playback. When speakers are combined in groups, you can stop playback for individual speakers but trying to restart playback again will fail.

I cannot get Siri to change the volume with "Set the volume of the Kitchen speakers to 20%" (where "Kitchen" is defined in the config file). Siri tells me it "wasn't able to find any devices in the Kitchen this time".

**Comment by adamstephenson** [comment]
@vanhoutem I've experienced the same thing with turn on/off of speakers in groups. I've had to removed the speaker from the group and re-add it for it to work again after it was 'stopped' via Siri. 

For the volume change, you may need to rename your kitchen speakers from "Kitchen" to "Kitchen Speakers" for Siri to be able to determine.

**Comment by JBX77** [comment]
You can create scenes with the Elgato Eve app to change the volume. I have scenes for 10, 20 and 30% which is enough for daily use.

**Comment by phylaxis** [comment]
I can't seem to control volume either. Is it just not supported or is there something wrong with my setup? I have not tried @JBX77's suggestion yet, though I expect that to work. I was hoping for more direct volume support though. Is it possible?

**Comment by miguelmolez** [comment]
Can this be set to start playing a song within an ap?
Like this for example to start playing a song with a command in a scene
http://www.automatedhome.co.uk/apple/smartthings-raspberry-pi-homebridge-siri-homekit-controlled-smart-home.html

**Comment by mattnewham** [comment]
Volume commands do not seem to work correctly -

Config

```        {
            "accessory": "Sonos",
            "name": "Landing Speakers",
            "room": "Landing"
        },

        {
            "accessory": "Sonos",
            "name": "Kitchen Speakers",
            "room": "Kitchen"
        },

        {
            "accessory": "Sonos",
            "name": "Living Room Speakers",
            "room": "Living Room"
```

Telling Siri to "Set the volume of the Speakers in the Landing to 80%" gets a response of "Sorry Matt, I was unable to find any speakerses in the Landing this time"

What is wrong with the setup?

**Comment by mattnewham** [comment]
For volume control see https://github.com/nfarina/homebridge-sonos/issues/3

**Comment by moochtm** [comment]
I see from [this site](https://developer.apple.com/library/ios/documentation/NetworkingInternet/Conceptual/HomeKitDeveloperGuide/AccessingServicesandTheirCharacteristics/AccessingServicesandTheirCharacteristics.html) that there is only a limited number of characteristics that Siri supports:

- Brightness
- Current door state
- Current temperature
- Lock state
- Power state
- Rotation speed
- Rotation direction
- Target door state
- Target state
- Target temperature

Is it possible that one these (e.g. Target state) could be set to an arbitrary string, and used to drive additional Sonos functionality? So, for example, this...

"Hey Siri, set the 'Kitchen Speakers' to 'Playing the Radio Station BBC Radio 4'"

...where 'Playing the Radio Station BBC Radio 4' is the arbitrary string, parsed and used to search for 'BBC Radio 4' and start playback if found?

**Comment by Phononaut** [comment]
Does anyone know how using the Boost factors in?  I have successfully connected to one of my Play: 3's but voice commands are not recognized at all.  I can change the volume and turn off/on thru the iOS app, but that's it.  I am wondering if I need to first connect Homebridge to the Boost?
